lib/model/query/submission-attachments.js

```diff
@@ -58,7 +58,7 @@ const _extractEncryptedAttachments = (ensure, submission, binaryFields, fileLook
       for (; i < names.length; i += 1) {
         if (names[i].isDefined()) {
           const name = names[i].get().trim();
-          const isClientAudit = (name === 'audit.csv');
+          const isClientAudit = (name === 'audit.csv.enc');
```

Comment:
Encrypted files always have an `.enc` suffix, right? Or should we be looking for `audit.csv` _or_ `audit.csv.enc`?

Reply:
All files with an encrypted submission including the submission xml are supposed to have a .enc suffix. So no need to look for audit.csv without suffix if the suffix isn’t explicitly stripped at that point.